var/spack/repos/builtin/packages/eccodes/package.py

```diff
@@ -45,7 +45,7 @@ class Eccodes(CMakePackage):
     git = "https://github.com/ecmwf/eccodes.git"
     list_url = "https://confluence.ecmwf.int/display/ECC/Releases"
 
-    maintainers("skosukhin")
+    maintainers("skosukhin", "victoria-cherkas", "dominichofer")
```

Comment:
Can you add me here as well, please?

Reply:
Sure, done: https://github.com/spack/spack/pull/41433/commits/a0bae1381a6d82c4ad931b98f3c336c465f32382